File: 2024/06/08/assets/arithmetic.py
#!/usr/bin/env python3
"""Do arithmetic operations."""
from sys import argv


def do_sum(operations: int) -> int:
    """Perform sum operations.

    Args:
        operations (int): The number of operations to perform.

    Returns:
        int: The series sum.
    """
    counter = 0
    for i in range(operations + 1):
        counter = counter + i
    return counter


def main():
    """Calculate a Fibonacci sequence number from user input."""
    term = int(argv[1])
    print(do_sum(term))


if __name__ == "__main__":
    main()

File: 2024/06/08/assets/arithmetic.js
#!/usr/bin/env node
/**
 * @module Do arithmetic operations.
 */

/**
 * Perform sum operations.
 *
 * @param {number} operations The number of operations to perform.
 * @returns {number} The series sum.
 */
function doSum(operations) {
    let sum = 0;
    for (let i = 0; i <= operations; i++) {
        sum = sum + i;
    }
    return sum;
}

const term = parseInt(process.argv[2]);
console.log(doSum(term));
